ArduCopter/Copter.h

```diff
@@ -607,17 +607,17 @@ class Copter : public AP_Vehicle {
     };
 
     static constexpr int8_t _failsafe_priorities[] = {
-                                                      Failsafe_Action_Terminate,
-                                                      Failsafe_Action_Land,
-                                                      Failsafe_Action_RTL,
-                                                      Failsafe_Action_SmartRTL_Land,
-                                                      Failsafe_Action_SmartRTL,
-                                                      Failsafe_Action_None,
+                                                      static_cast<int8_t>(Failsafe_Action::Terminate),
```

Comment:
I wonder if we should take this opportunity to have these indented further to the left?

Reply:
No. It's nicely aligned on it's parent bracket :)

Reply:
![image](https://user-images.githubusercontent.com/7077857/75722365-2b9fe780-5d2e-11ea-9cee-12f8847a31bc.png)

There we go - immortalised :-P